Backend/src/main/java/lt/vu/ads/models/Address.java

```diff
@@ -4,25 +4,33 @@
 import lombok.Getter;
 import lombok.Setter;
 
+import javax.persistence.*;
+import java.util.List;
+
 @Getter
 @Setter
+@Table
+@Entity
 public class Address {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false)
     private Long id;
 
+    @Column(nullable = false)
     private String city;
 
+    @Column(nullable = false)
     private String street;
 
+    @Column(nullable = false)
```

Comment:
Maybe Flat number is needed (which is optional)?

Reply:
you can write flat number in house number. for example, house number - flat number